dev/com.ibm.websphere.appserver.features/visibility/public/mpFaultTolerance-3.0/com.ibm.websphere.appserver.mpFaultTolerance-3.0.feature

```diff
@@ -10,7 +10,6 @@ IBM-ShortName: mpFaultTolerance-3.0
 Subsystem-Name: MicroProfile Fault Tolerance 3.0
 -features=com.ibm.websphere.appserver.org.eclipse.microprofile.faulttolerance-3.0, \
  com.ibm.websphere.appserver.cdi-2.0, \
- com.ibm.websphere.appserver.concurrent-1.0,\
```

Comment:
You should be able to remove this buildpath dependency from other fault tolerance projects as well except for the ft.cdi_fat project.

Reply:
Thanks, done